contracts/utils/RFQLibEIP712.sol

```diff
@@ -56,12 +56,12 @@ library RFQLibEIP712 {
         );
     }
 
-    bytes32 public constant FILL_WITH_PERMIT_TYPEHASH = 0x4ea663383968865a4516f51bec2c29addd1e7cecce5583296a44cc8d568cad09;
+    bytes32 public constant FILL_TYPEHASH = 0x9b822fe3212f0c2a34b120a750281383593d621b7cee5888ef0bdc840bf6ff2c;
```

Comment:
This refactor seems to affect current clients. Should we leave it as before until we decide to do a breaking upgrade?